.config/dictionaries/project.dic

```diff
@@ -76,6 +79,7 @@ fontello
 formz
 Formz
 fuzzer
+gafhhkghbfjjkeiendhlofajokpaflmk
```

Comment:
You're sure this is a valid word? Probably would be better just to disable it for the lines that break the cspell, i.e.:

```
/* cSpell:disable */
final myVar = 'gafhhkghbfjjkeiendhlofajokpaflmk';
/* cSpell:enable */
```